src/Message.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace ResellerClub;
+
+class Message
+{
+    /**
+     * @var string
+     */
+    protected $message;
+
+    /**
+     * Create a new message.
+     *
+     * @param string
+     */
+    public function __construct(string $message)
+    {
+        $this->message = $message;
+    }
+
+    /**
+     * @return string
```

Comment:
Can you put in a description here, even if it's only `Get the string representation of the object.`

Reply:
Done